common/changes/miwhea-fabric-core-6_2017-03-21-16-51.json

```diff
@@ -0,0 +1,20 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Update to Fabric Core 6.0.1",
+      "type": "major"
+    },
+    {
+      "packageName": "@uifabric/utilities",
+      "comment": "Update to Fabric Core 6.0.1",
+      "type": "patch"
+    },
+    {
+      "packageName": "@uifabric/example-app-base",
+      "comment": "Update to Fabric Core 6.0.1",
+      "type": "major"
```

Comment:
I'm not sure if this should be major or just a patch. Do we think people are using example-app-base as the foundation for an app, and then referencing Fabric Core variables and mixins from within `node_modules`? If it's just for a demo I think this could be a patch.